### DISCUSSION TITLE
Switch between shared and public Key encryptation ?

### DISCUSSION THREAD

**Discussion by F8GHE** [meshtastic/firmware#5374] [General]
Is there a way to manually switch from "shared Key" to "public Key" encryption for direct messages?
![Screenshot_20241117-111318](https://github.com/user-attachments/assets/d953b34a-6d2b-42af-b43b-91d435563cae)

73s F8GHE Christian

**Comment by jcyrio** [comment]
I think this would be a useful feature. I haven't switched to 2.5 yet because my MQTT backend doesn't work with the DMs on 2.5. I've heard of others doing the same. I'm using only private encrypted channels so the DM encryption isn't useful to me.
